services/provider/client/client.go

```diff
@@ -142,7 +142,15 @@ const (
 	StorageTypeSharedfilesystem
 )
 
-func (cc *OCSProviderClient) FulfillStorageClassClaim(ctx context.Context, consumerUUID, storageClassClaimName, encryptionMethod string, storageType StorageType) (*pb.FulfillStorageClassClaimResponse, error) {
+func (cc *OCSProviderClient) FulfillStorageClassClaim(
+	ctx context.Context,
+	consumerUUID,
+	storageClassClaimName,
+	encryptionMethod,
+	storageProfile string,
```

Comment:
nit: I would prefer if it was added in the end

Reply:
this was done to keep the strings args in a group or else i need to separate out the group of string. Please let me know what do you think